extend.sh

```diff
@@ -422,14 +424,43 @@ function extend_boot_partition() {
     fi
 }
 
+function activate_volume_group(){
```

Comment:
`pvs` and `vgchange` - use the full path

Reply:
Done in both instances (`deactivate` and `activate volume group` functions)